### PR TITLE
Urp/depth priming tests

### PR DESCRIPTION
**Pull request by NicoLeyman**
### Purpose of this PR
The recently landed PR adding Depth Priming didn't contain any automated test coverage. This PR aims to correct that by adding a new graphics test.
Since this test ran into an issue related to the recent swapbuffer changes this also contains a fix for that to allow the test to succeed.

---
### Testing status
New graphics test added. Ref images are being added.

image

The test shows a set of boxes rendered with a camera stack, splitscreen rendering. Since Depth Priming can have fallout I opted to use some of the existing materials on the scene objects. SSAO and transparent materials are included and definitely need to stay part of the test.

---
### Comments to reviewers
Code changes provided by Robin.

### CODE REVIEW

File: com.unity.render-pipelines.universal/Runtime/RenderTargetBufferSystem.cs [hunk 1]
```diff
@@ -29,11 +29,6 @@ public RenderTargetBufferSystem(string name)
             RTB.Init(name + "B");
         }
 
-        public RenderTargetHandle GetBackBuffer()
-        {
-            return m_FirstIsBackBuffer ? RTA : RTB;
-        }
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Runtime/RenderTargetBufferSystem.cs:35]: Note for RM: this class is internal, not breaking change to user.

File: com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs [hunk 2]
```diff
@@ -389,12 +389,14 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
                 m_DeferredLights.IsOverlay = cameraData.renderType == CameraRenderType.Overlay;
             }
 
+            CommandBuffer cmd = CommandBufferPool.Get();
```
[the remark this answers is not in the document]
Comment (phi-lira) [reply com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs:392]: Dont' we have already a `cmd` available in `UniversalRenderer`?

File: TestProjects/UniversalGraphicsTest_Foundation/ProjectSettings/TagManager.asset [hunk 3]
```diff
@@ -17,8 +17,8 @@ TagManager:
   - VolumeLayer2
   - VolumeLayer3
   - VolumeLayer4
-  - 
-  - 
+  - Terrain
+  - Props
```
Comment (phi-lira) [inline TestProjects/UniversalGraphicsTest_Foundation/ProjectSettings/TagManager.asset:21]: Why is it needed?

Comment (NicoLeyman) [reply TestProjects/UniversalGraphicsTest_Foundation/ProjectSettings/TagManager.asset]: It's not, I've changed it to just use the existing UI tag so I don't need to any new ones.

File: com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (robin-demoor) [reply com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs]: Turns out we won't need the cmd in the UniversalRenderer Setup so this is all removed

**Review by simon-engelbrecht-soerensen — APPROVED** [review]
Would be nice to have screenshots of ref images that were added in the PR.
Otherwise looks good.

**Review by phi-lira — APPROVED** [review]

**Pull request merged**